src/theme/dark.ts

```diff
@@ -77,7 +77,7 @@ const colors = {
   },
 };
 
-const global = {
+const globalStyle = {
```

Comment:
After upgrading the `target` to `es6` in `tsconfig.json`. I got the error when running `npm run build:release`:
> SyntaxError: Identifier 'global' has already been declared

`global` is a reserved keyword in JavaScript. We should avoid declaring it. So I have to rename it.